drafts/concise_guide.ipynb

```diff
@@ -92,7 +92,66 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When creating a response, FastHTML will automatically render tags using the to_xml function. "
+    "This example demonstrates key aspects of how FastHTML components handle attributes:\n",
+    "\n",
+    "```python\n",
+    "Label(\n",
+    "    \"Choose an option\", \n",
+    "    Select(\n",
+    "        Option(\"one\", value=\"1\", selected=True),  # Boolean True renders just the attribute name\n",
+    "        Option(\"two\", value=\"2\", selected=False), # Boolean False omits the attribute entirely\n",
+    "        Option(\"three\", value=3),                 # Non-string values are converted to strings\n",
+    "        cls=\"selector\",                           # 'cls' becomes 'class' in HTML (Python reserved word)\n",
+    "        _id=\"counter\",                            # Leading underscore is stripped during rendering\n",
+    "        **{'@click':\"alert('Clicked');\"},         # Dict unpacking for attributes with special characters\n",
+    "    ),\n",
+    "    _for=\"counter\",                               # '_for' becomes 'for' (can also use 'fr')\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "FastHTML components can be linked by matching `_id` with `_for` attributes. The library handles Python reserved words by using alternatives like `cls` for `class` and stripping leading underscores during rendering."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can create new FT components in two ways, the preferred approach is to import the new component from `fasthtml.component`. If the component doesn't exist within that module, FastHTML will create it."
```

Comment:
No need to mention two ways if we can get away with one! :) Also the comment and the markdown overlap a bit - try to remove the comment so info is in one place.